app/src/main/java/com/kickstarter/services/apiresponses/ErrorEnvelope.kt

```diff
@@ -32,12 +32,12 @@ class ErrorEnvelope private constructor(
         @Parcelize
         data class Builder(
             private var id: Long = 0L,
-            private var name: String? = null,
-            private var email: String? = null
+            private var name: String = "",
+            private var email: String = ""
         ) : Parcelable {
             fun id(id: Long?) = apply { this.id = id ?: 0L }
-            fun name(name: String?) = apply { this.name = name }
-            fun email(email: String?) = apply { this.email = email }
+            fun name(name: String?) = apply { this.name = name?.let { it } ?: "" }
+            fun email(email: String?) = apply { this.email = email?.let { it } ?: "" }
```

Comment:
Small syntax comment:  you can do `name ?: "" `, you don't need the let in this situation